expression/constant_test.go

```diff
@@ -56,14 +56,22 @@ func newLonglong(value int64) *Constant {
 }
 
 func newDate(year, month, day int) *Constant {
+	return newTimeConst(year, month, day, 0, 0, 0, mysql.TypeDate)
+}
+
+func newTimestamp(yy, mm, dd, hh, min, ss int) *Constant {
+	return newTimeConst(yy, mm, dd, hh, min, ss, mysql.TypeTimestamp)
```

Comment:
The arguments `hh, min, ss` dosen't use in the `newTimeConst` function, why still parse them into the function??

Reply:
A timestamp value has the `hh, min, ss` part, while a date value contains only the `yy, mm, dd` information. That's why the `newDate` function has fewer parameters than `newTimestamp`.

We pass 0 as arguments, but it doesn't means `hh, min, ss` would never be useful in the future.
@Fatpa